Failed to publish artifacts

```
* What went wrong:
Execution failed for task ':grails-redis:groovydoc'.
> Could not resolve all files for configuration ':grails-redis:documentation'.
   > Could not resolve com.google.guava:guava:32.1.3-jre.
     Required by:
         project :grails-redis > com.github.javaparser:javaparser-core:3.25.7
      > Cannot choose between the following variants of com.google.guava:guava:32.1.3-jre:
          - androidRuntimeElements
          - jreRuntimeElements
        All of them match the consumer attributes:
          - Variant 'androidRuntimeElements' capabilities com.google.collections:google-collections:32.1.3-jre and com.google.guava:guava:32.1.3-jre:
              - Unmatched attributes:
                  - Provides org.gradle.category 'library' but the consumer didn't ask for it
                  - Provides org.gradle.dependency.bundling 'external' but the consumer didn't ask for it
                  - Provides org.gradle.jvm.environment 'android' but the consumer didn't ask for it
                  - Provides org.gradle.jvm.version '8' but the consumer didn't ask for it
                  - Provides org.gradle.libraryelements 'jar' but the consumer didn't ask for it
                  - Provides org.gradle.status 'release' but the consumer didn't ask for it
                  - Provides org.gradle.usage 'java-runtime' but the consumer didn't ask for it
          - Variant 'jreRuntimeElements' capabilities com.google.collections:google-collections:32.1.3-jre and com.google.guava:guava:32.1.3-jre:
              - Unmatched attributes:
                  - Provides org.gradle.category 'library' but the consumer didn't ask for it
                  - Provides org.gradle.dependency.bundling 'external' but the consumer didn't ask for it
                  - Provides org.gradle.jvm.environment 'standard-jvm' but the consumer didn't ask for it
                  - Provides org.gradle.jvm.version '8' but the consumer didn't ask for it
                  - Provides org.gradle.libraryelements 'jar' but the consumer didn't ask for it
                  - Provides org.gradle.status 'release' but the consumer didn't ask for it
                  - Provides org.gradle.usage 'java-runtime' but the consumer didn't ask for it

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org/

BUILD FAILED in 11s
7 actionable tasks: 6 executed, 1 from cache
```

--- a/plugin/build.gradle
+++ b/plugin/build.gradle
@@ -61,7 +61,9 @@ dependencies {
     api 'redis.clients:jedis:5.1.0'
     api 'com.google.code.gson:gson:2.10.1'
 
-    documentation 'com.github.javaparser:javaparser-core:3.25.7'
+    documentation('com.github.javaparser:javaparser-core:3.25.7') {
+        transitive = false
+    }
 }
 
 groovydoc {

